assets/list.js

```diff
@@ -0,0 +1,48 @@
+function makeUL(array) {
+    var list = document.createElement('ul');
+    for (var i = 0; i < array.length; i++) {
+        var item = document.createElement('li');
+        item.appendChild(array[i]);
+        list.appendChild(item);
+    }
+    return list;
+}
+// get list of events
+var eventList = Array.from(document.getElementsByClassName('event-link'))
+var now = new Date()
+// filter to get upcoming list
+var upcomingList = eventList.filter((a) => new Date(a.attributes.date.value.split(' ')[0] + ' ' + a.attributes.date.value.split(' ')[1]) >= new Date())
+var eventDiv = document.getElementById('event-list')
+var upcomingEventDiv = document.getElementById('upcoming-event')
+// this is the events page
```

Comment:
Slight (nit-picky) preference to capitalize, and have some closing punctuation (':', or '.', or '...')